lib/quicly.c

```diff
@@ -2896,16 +2896,16 @@ static int server_collected_extensions(ptls_t *tls, ptls_handshake_properties_t
     ack_frequency_set_next_update_at(conn);
 
     /* update UDP max payload size to:
-     * max(current, min(max_the_remote_sent, remote.tp.max_udp_payload_size, local.tp.max_udp_payload_size)) */
+     * min(remote.tp.max_udp_payload_size, max(current, max_the_remote_sent, local.tp.max_udp_payload_size)) */
```

Comment:
This change is technically correct, but we also fix the doc-comment on `quicly_context_t::initial_egress_max_udp_payload_size`.

Considering that, how about changing this block comment to:

```suggestion
/* update UDP max payload size, as described in the doc-comment of `quicly_context_t::initial_egress_max_udp_payload_size` */
```
and updating that doc-comment accordingly?

Reply:
Yes, good call, thank you. Done in 8ff994fa9836cf5da62c0c73d574ff3ede06b0f7